connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -648,7 +636,6 @@ private static InputStream getInputStream(final CloseableHttpResponse response)
             @Override
             public void close() throws IOException {
                 response.close();
-                super.close();
             }
```

Comment:
See httpclient documentation. To properly dispose of resources, one must either close the CloseableHttpResponse or the InputStream. Closing both is unnecessary and causes an exception in httpclient 4.5.2.